### PR TITLE
Add multiple scheduler docs

### PR DESCRIPTION
**Pull request by Poor12**
Signed-off-by: Poor12 <shentiecheng@huawei.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add multiple scheduler docs

**Which issue(s) this PR fixes**:
Part of https://github.com/karmada-io/karmada/issues/3024

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/developers/customize-karmada-scheduler.md [hunk 1]
```diff
@@ -187,4 +187,167 @@ kubectl kubectl --kubeconfig ~/.kube/karmada.config --context karmada-host edit
...
+            httpGet:
+              path: /healthz
+              port: 10351
+              scheme: HTTP
+            failureThreshold: 3
+            initialDelaySeconds: 15
+            periodSeconds: 15
+            timeoutSeconds: 5
+          command:
+            - /bin/karmada-scheduler
```
Comment (RainbowMango) [inline docs/developers/customize-karmada-scheduler.md:235]: Does the customized scheduler also named `karmada-scheduler`?

File: docs/developers/customize-karmada-scheduler.md [hunk 2]
```diff
@@ -187,4 +187,167 @@ kubectl kubectl --kubeconfig ~/.kube/karmada.config --context karmada-host edit
...
+            - --kubeconfig=/etc/kubeconfig
+            - --bind-address=0.0.0.0
+            - --secure-port=10351
+            - --enable-scheduler-estimator=true
+            - --leader-elect-resource-name=my-scheduler
+            - --scheduler-name=my-scheduler
```
Comment (RainbowMango) [inline docs/developers/customize-karmada-scheduler.md:241]: We should highlight the key flags that a customized scheduler should take care of.

File: docs/developers/customize-karmada-scheduler.md (hunk 1 above)
Comment (Poor12) [reply docs/developers/customize-karmada-scheduler.md:235]: This is the name of the compiled binary. Changing to `karmada-scheduler` will not affect the custom scheduler logic. The same as https://kubernetes.io/zh-cn/docs/tasks/extend-kubernetes/configure-multiple-schedulers/#%E4%B8%BA%E8%B0%83%E5%BA%A6%E5%99%A8%E5%AE%9A%E4%B9%89-kubernetes-deployment.

**Review by RainbowMango — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**